tutor/src/models/task-plans/teacher/scores.js

```diff
@@ -82,6 +108,10 @@ class TaskPlanScoreHeading extends BaseModel {
     return 'Tutor' !== this.type;
   }
 
+  @computed get displayType() {
+    return this.type === 'FR' ? 'WRQ' : this.type;
+  }
+
```

Comment:
looks the `FR` is coming from the backend, I just made https://github.com/openstax/tutor-server/pull/2067 to fix it so we can remove this